server/routes/routes.js

```diff
@@ -0,0 +1,100 @@
+const Product = require('Product model goes here');
```

Comment:
given that this file is under the 'routes' folder and links all api routes, we could call it `/routes/index.js`